index.js

```diff
@@ -109,43 +123,24 @@ async function main() {
       core.info(`All rows will be purged!`);
     }
 
-    const releasesToPurge = [];
-    for (let i = 0; i < appReleases.length; i++) {
-      if (i >= toKeep) {
-        releasesToPurge.push(appReleases[i]);
-      }
-    }
+    // Only delete releases after the number to keep
+    const releasesToPurge = appReleases.slice(0, -toKeep);
```

Comment:
Does this slice keep the head of the array or the tail? We want to keep the head (as these are the latest versions).

Reply:
Yep, this will remove the last `toKeep` items from the array - which I have just realised is incorrect. Pushed a changed 🙏